src/pykeen/pipeline/api.py

```diff
@@ -1076,6 +1076,14 @@ def pipeline(  # noqa: C901
         ),
     )
 
+    # Log loss parameters
+    _result_tracker.log_params(
+        params=dict(
+            loss_kwargs=loss_kwargs
+        ),
```

Comment:
Maybe we should also log the loss' class
```suggestion
        params=dict(
            loss=loss_resolver.normalize_inst(model.loss),
            loss_kwargs=loss_kwargs,
        ),
```

Reply:
I initially also had loss (class) as a param, but i realized that there already exists a model_kwargs.loss logged, so i decided to not "duplicate" that information.